src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/blocklistentryeditors/inlineblock/inlineblock.editor.controller.js

```diff
@@ -0,0 +1,25 @@
+(function () {
+    'use strict';
+
+    function InlineBlockEditor($scope) {
+
+        const bc = this;
+
+        bc.openBlock = function(block) {
+
+            // if we are closing:
+            if (block.active === true) {
+                // boardcast the formSubmitting event to trigger syncronization or none-live property-editors
+                $scope.$broadcast("formSubmitting", { scope: $scope });
+                // Some property editors need to performe an action after all property editors have reacted to the formSubmitting.
+                $scope.$broadcast("postFormSubmitting", { scope: $scope });
```

Comment:
If methods executed by ´formSubmitting´ executes immediately, then Yes. We do not have an architecture that allows somethings to take time.

Reply:
OK so this could be a problem then & catch us out. Is this the right solution then @Shazwazza & @nielslyngsoe or is this the best we can ever do for this?